### PR TITLE
Add WebGL extensions

### PR DESCRIPTION
**Pull request by Elchi3**
https://github.com/web-platform-dx/web-features/pull/1583 decided to put some WebGL extensions in the main WebGL feature but not all of them. I think no WebGL extension should be with the main WebGL feature. The extensions have very different support, some are meant to be for certain operating systems only (mobile only, for example), some require specific drivers, etc. I think it is best if they stand on their own. I introduced a "webgl-extensions" group to keep them together.

Still to be done are the descriptions, but would be nice to get a 👍 on the approach here before writing all of them.

### CODE REVIEW

**Comment by Elchi3** [comment]
@ddbeck are you okay with the "The `EXT_color_buffer_half_float` extension to WebGL TODO" structure for the descriptions here? (I will need to provide the rest of the sentences)

And for the name, I now have "WebGL OES_texture_float extension". I'm assuming you would rather like "OES_texture_float WebGL extension"?

**Comment by ddbeck** [comment]
Yeah, I think that looks good.

> I'm assuming you would rather like "OES_texture_float WebGL extension"?

Yes, please! Though I imagine "OES_texture_float" or "EXT_color_buffer_half_float" is sufficiently unique by itself.

To embellish on the descriptions a little, I'd write the template like this:

> The EXT_[var] extension to WebGL [verbs] [nouns]…

This would also be fine:

> The EXT_[var] WebGL extension [verbs] [nouns]…

Though I expect that's a best case scenario. I won't be bothered to see some "extension represents a…" or "is a…". The audience for this is pretty niche and I won't lose sleep over some basic descriptions, if it's obvious when someone who doesn't know anything about WebGL realizes they're not the audience for this information. 😄

**Comment by Elchi3** [comment]
Thanks Daniel. This is now ready to review and https://github.com/web-platform-dx/web-features/commit/d1449156522a9c0714d2ab055b32b0abdbb9bd94 might help for  taking a look at names and descriptions specifically.

File: features/ext-texture-compression-rgtc.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: EXT_texture_compression_rgtc WebGL extension
+description: The `EXT_texture_compression_rgtc` extension for WebGL 1.0 and 2.0 contexts adds RGTC compressed texture formats. RGTC is a block-based texture compression format suited for unsigned and signed red and red-green textures (Red-Green Texture Compression).
```
Comment (ddbeck) [inline features/ext-texture-compression-rgtc.yml:2]:
```suggestion
description: The `EXT_texture_compression_rgtc` extension for WebGL 1.0 and 2.0 contexts adds RGTC compressed texture formats. RGTC is a block-based texture compression format suited for unsigned and signed red and red-green textures. Also known as Red-Green Texture Compression.
```

File: features/ext-texture-norm16.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: EXT_texture_norm16c WebGL extension
+description: The `EXT_texture_norm16` extension to WebGL 2.0 contexts adds 16-bit signed normalized and unsigned normalized formats (fixed-point texture, renderbuffer and texture buffer).
```
Comment (ddbeck) [inline features/ext-texture-norm16.yml:2]:
```suggestion
description: The `EXT_texture_norm16` extension to WebGL 2.0 contexts adds 16 bit signed normalized and unsigned normalized fixed-point texture, renderbuffer, and texture buffer formats.
```

File: features/oes-texture-half-float.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: OES_texture_half_float WebGL extension
+description: The `OES_texture_half_float` extension for WebGL 1.0 contexts adds texture formats with 16- (aka half float) and 32-bit floating-point components.
```
Comment (ddbeck) [inline features/oes-texture-half-float.yml:2]:
```suggestion
description: The `OES_texture_half_float` extension for WebGL 1.0 contexts adds texture formats with 16-bit (also known as half float) and 32-bit floating-point components.
```

**Review by ddbeck — APPROVED** [review]
OK, I think this is ready to go. A few comments cleaning up the descriptions here. Please accept what you like, modify (or ignore) what you don't, and merge it. Thank you!

**Pull request merged**